libvcx/src/api_lib/api_c/vcx.rs

```diff
@@ -769,7 +769,7 @@ pub mod test_utils {
 }
 
 #[cfg(test)]
-#[allow(unused_imports)]
+#[allow(unused_imports)] // FIXME: remove it
```

Comment:
We are using TODO (or todo) instead of FIXME in this repository (being consistent makes it easier to find all TODOs)

Reply:
I've pushed the fix